.github/workflows/software-environments-nightly.yml

```diff
@@ -0,0 +1,76 @@
+name: Create software environments
+
+on:
+  schedule:
+    # Runs at 01:00 every day after pre-release
+    - cron: "0 1 * * *"
+
+jobs:
+  build:
+    name: Python ${{ matrix.python-version }}, nightly build, ${{ matrix.server }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+        server: ["https://staging.coiledhq.com"]
+        include: #leaving this logic as we build nightly soft env for prod. 
```

Comment:
I don't quite understand the meaning behind this comment -- could you elaborate a bit more?

Reply:
Yes, I think it's not clear. But we talked about the possibility of having nightlies soft-envs in prod and this comment meant to say, leave this logic in case we want to add in the future the other server.